docs/cep/tutorials/executing-scripts.md

```diff
@@ -4,9 +4,9 @@ sidebar_position: 10
 
 # Executing Scripts
 
-The script provides the ability to write custom functions in other programming languages and execute them within Stream applications. The custom functions using scripts can be defined via the function definitions and accessed in queries similar to any other inbuilt functions.
+The script provides the ability to write custom functions in other programming languages and execute them within Stream Workers. The custom functions using scripts can be defined via the function definitions and accessed in queries similar to any other inbuilt functions.
```

Comment:
```suggestion
The script provides the ability to write custom functions in other programming languages and execute them within stream workers. The custom functions using scripts can be defined via the function definitions and accessed in queries similar to any other inbuilt functions.
```

Reply:
Do not capitalize unless there is a reason, like you are referencing something in the UI and matching its capitalization. Or if it's a heading, in which case it should be title case.